example/lib/app/app.dart

```diff
@@ -133,6 +133,8 @@ class _ChewieDemoState extends State<ChewieDemo> {
               ),
       ),
 
+      hideControlsTimer: const Duration(seconds: 1),
```

Comment:
We can add a variable: 

```dart
({
... // Your variable
this.createControlsTimer = const Duration(seconds: 1),
)};
```

```dart
final Duration createControlsTimer;
```

It would be better if the developer is able to modify it.